sanghyeok/양꼬치.js

```diff
@@ -0,0 +1,8 @@
+function solution(n, k) {
+    if(n<10){
+    //n(양꼬치)이 10게 미만 주분 시 n과 음료수값(k) 은 그대로 출력)    
+        return (n * 12000) + (k * 2000)  
+    }
+    //n이 10 이상일 경우 10개당 k의 기격에서 할인
+    return (n * 12000) + (k * 2000) - (Math.floor(n/10) * 2000)
```

Comment:
```suggestion
    return (n * 12000) +  2000 * (k - Math.floor(n/10))
```
2000으로 묶어서 곱해줘도 괜찮을 것 같아요~!!
10개 미만도 고려하셨군요!! 저는 그 부분 생각을 못했네요 🫢
덕분에 포인트 잡아서 가봅니다~!! 👍

Reply:
아하 저는 단순히 구현만 생각하느라고 가독성 여부를 확인하지 못했네요 피드백 감사합니다!!